Changelog.md

```diff
@@ -3,6 +3,7 @@
 Bugfixes:
  * Code generator: Defensively pad allocation of creationCode and runtimeCode to multiples of 32 bytes.
  * Parser: Disallow empty import statements.
+ * Type Checker: Fix internal error when a struct array index is not an unsigned.
```

Comment:
```suggestion
 * Type Checker: Fix internal error when a struct array index is not an unsigned integer.
```

Reply:
or
```suggestion
 * Type Checker: Fix internal error when a struct array index does not fit into an uint256.
```

Reply:
"a uint" :)

Reply:
Mind you all, I just copied this change log entry from https://github.com/ethereum/solidity/pull/5195/files#diff-9b8fab691c00b9e5380b19ce882f3271R128
and added "struct" :P